core/chains/evm/gas/models_test.go

```diff
@@ -1 +1,90 @@
-package gas
+package gas_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/assets"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/gas"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/gas/mocks"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWrappedEvmEstimator(t *testing.T) {
```

Comment:
Can we add `t.Parellel()` here? I like adding it wherever possible in order to save future pain trying to optimize test run time